src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -74,11 +75,35 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <PropertyGroup> <!-- Can be removed once runtime repo switches over to 6.0 drop of the SDK repo -->
+      <JitTargetArchComponent>$(TargetArchitecture)</JitTargetArchComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'windows'">win</JitTargetOSComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'linux'">unix</JitTargetOSComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'osx'">osx</JitTargetOSComponent>
+      <JitBuildArchComponent>$(BuildArchitecture)</JitBuildArchComponent>
+      <JitArchSuffix>$(JitTargetArchComponent)_$(JitBuildArchComponent)</JitArchSuffix>
+      <JitFileName>clrjit_$(JitTargetOSComponent)_$(JitArchSuffix)</JitFileName>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(TargetOS)' == 'osx'">
+      <JitFileNameOSX>clrjit_unix_osx_$(JitArchSuffix)</JitFileNameOSX>
+      <JitFileName Condition="Exists($(JitFileNameOSX))">$(JitFileNameOSX)</JitFileName>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <JitExtension>.so</JitExtension>
+      <JitExtension Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">.dylib</JitExtension>
+      <JitExtension Condition="'$(OS)' == 'Windows_NT'">.dll</JitExtension>
```

Comment:
Can we use `[MSBuild]::IsOsPlatform('OSX')` instead? See https://github.com/dotnet/runtimelab/blob/feature/NativeAOT/src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj#L24-L27

Reply:
Removed in 2nd commit.